clients/js/src/index.ts

```diff
@@ -66,8 +66,9 @@ let CohereAiApi: any;
 
 export class CohereEmbeddingFunction {
   private api_key: string;
+  private model: string;
 
-  constructor(cohere_api_key: string) {
+  constructor(api_key: string, model?: string) {
```

Comment:
sorry one more change here - we need to set this to `cohere_api_key` to avoid a breaking change.

Reply:
forsure